A community template on using AI in Quantum Chemistry applications

karsar opened the pull request:
This template provides a slightly non-computer-science example. One of the important topics in modern quantum chemistry is the acceleration of computations. To this end, AI techniques, such as the MACE package implementing neural network potentials, are used for calculating energies and forces.

While MACE-type equivariant neural networks excel in prediction accuracy, their application to molecular dynamics simulations reveals a less understood connection between the specific architecture of the MACE family and the success of molecular dynamics simulations. This slightly, but not fully, toy example sets the stage for an open-ended study.

An example of a generated paper can be found [https://drive.google.com/file/d/1dO8qXeCWaXkqP-sbV4z1S5RwY--pmggh/view?usp=sharing](url)

Three extra packages are required:
`pip install mace-torch`
`pip install MDAnalysis`
`pip install statsmodels`

conglu1997 commented:
Hi, thanks for the contribution! Would it be possible to avoid adding the many template-specific files directly but add instructions on how to download them in a README in the template folder? Cheers!

karsar commented:
Thanks for the suggestions. I've moved the datasets (they're tiny) to a different GitHub repository and explained how to download them, along with all the other required steps, in a README file. Let me know if there's anything else to improve.

conglu1997 commented on templates/MACE/ideas.json:
```diff
@@ -0,0 +1,38 @@
+[
+    {
+        "Name": "number of interactions",
```
Names should have _ in them!

karsar replied:
corrected

conglu1997 commented on templates/MACE/latex/fancyhdr.sty:Zone.Identifier:
erroneous files?

karsar replied:
somehow WSL2 produced them, deleted now

conglu1997 commented on templates/MACE/latex/template.tex:
```diff
@@ -0,0 +1,194 @@
+
+@inproceedings{
+edm,
+title={Elucidating the Design Space of Diffusion-Based Generative Models},
```
inappropriate base cites for this paper!

karsar replied:
full response below

conglu1997 reviewed (commented):
I have left a few comments!

karsar commented:
Yes, indeed, there were erroneous files. I've deleted them and renamed the ideas. The LaTeX template was mistakenly taken from a diffusion experiment without any changes to the base references. I was running my experiments using the incorrect template. Funny enough, all of the citations in the final text of the "papers" produced were correct, except for one citing a review on diffusion models. I reran the experiment with a corrected LaTeX template and updated the link to the example paper in README.md.

conglu1997 commented:
Tysm!
